.github/rebase.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+source $(dirname "${BASH_SOURCE[0]}")/bot-pr-comment-base.sh
+
+if [[ "$PR_MERGED" == "true" ]]; then
+  bot_error "PR already merged!"
+fi
+
+git remote add base "$BASE_URL"
```

Comment:
In `format.sh` this is `origin`. Either are fine with me, but maybe picking one would be better.

Reply:
The reason for this is slightly convoluted: Due to the flat checkout `fetch-depth: 0` the branch `origin/develop` doesn't contain any history except for the top commit, so I needed to pull the base history manually, so the rebase works out of the box. `format.sh` on the other hand only needs `origin/develop` for the formatting files, the actual commits happen on the fork. The corresponding lines in `format.sh` are probably unnecessary.

Reply:
Ah I see, that makes more sense suddenly, thanks for the explanation!